packages/web3-wallet/package.json

```diff
@@ -40,6 +40,7 @@
     "@types/node": "^16.18.23",
     "bip32": "3.1.0",
     "bip39": "3.0.4",
+    "buffer": "^6.0.3",
```

Comment:
How about using a fixed version for this one to mitigate supply chain attacks, since it's a critical dependency for cryptography?

Reply:
done